Validation/Configuration/python/globalValidation_cff.py

```diff
@@ -120,11 +120,15 @@
 )
 globalValidationTrackingOnly = cms.Sequence()
 
-def _modifyGlobalValidationForPhase2( theProcess ):
-    theProcess.load('Validation.Configuration.gemSimValid_cff')
-    theProcess.load('Validation.Configuration.me0SimValid_cff')
-    theProcess.globalValidation += theProcess.gemSimValid
-    theProcess.globalValidation += theProcess.me0SimValid
+from Validation.Configuration.gemSimValid_cff import *
+from Validation.Configuration.me0SimValid_cff import *
+
+_run3_globalValidation = globalValidation.copy()
+_run3_globalValidation += gemSimValid
+
+_phase2_globalValidation = _run3_globalValidation.copy()
+_phase2_globalValidation += me0SimValid
 
 from Configuration.StandardSequences.Eras import eras
-modifyConfigurationStandardSequencesGlobalValidationForPhase2_ = eras.phase2_muon.makeProcessModifier( _modifyGlobalValidationForPhase2 )
+eras.run3_GEM.toReplaceWith( globalValidation, _run3_globalValidation )
+eras.phase2_muon.toReplaceWith( globalValidation, _phase2_globalValidation )
```

Comment:
@Dr15Jones Now I'm confused even more. Has the implementation of `cms.Modifier` changed recently (doesn't look so)?

Here is my understanding of the (current) internals of the era system. The fact that `Phase2` `ModifierChain` is (essentially) `(phase2_muon, run3_GEM)` means only that the sub-eras are activated (their `__chosen` member is set to `True`) in that order, and this has nothing to do with the execution order of various `toModify()` and `toReplaceWith()` calls (which both seem to still do their magic immediately). Therefore I'd expect the order of applied modifications to be dictated by the order of `toModify()` and `toReplaceWith()` calls in the code. In the code above, this order is `run3_GEM` first, then `phase2_muon`. In case both being active, I'd expect the `globalValidation` to be the `_phase2_` one.

It also looks to me that the intention is to have in
* `run3_GEM` default globalValidation + `gemSimValid`
* `phase2_muon` default globalValidation + `gemSimValid` + `me0SimValid`

To me it looks like the configuration above would do what is expected. But maybe I've missed some detail.